lib_device_control/host/control_host.h

```diff
@@ -1,12 +1,13 @@
-// Copyright 2016-2021 XMOS LIMITED.
+// Copyright 2016-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 #ifndef __control_host_h__
 #define __control_host_h__
 
 #include "control.h"
-#include "control_transport.h"
+#include "control_shared.h"
```

Comment:
shouldn't need this as control.h includes control_shared.h?

Reply:
Good point. I will remove that include.